eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -2,7 +2,7 @@ parameters:
   Repos: []
   DirectoryToSync: eng/common
   CommitMessage: commit-message-not-set
-  PRBranchName: branch-name-not-set
+  UpStreamBranchName: branch-name-not-set
```

Comment:
Why change the name here?

Reply:
To better reflect whats going on since we no longer create the pull request using that parameter.